lib/enbala/vpp.ex

```diff
@@ -1,15 +1,18 @@
 defmodule Enbala.Vpp do
   @moduledoc """
-  A Virtual Power Plant or VPP is a collection of controllable assets, batteries in our
-  case, that we can use to bring additional stability to the grid.
+  A Virtual Power Plant (VPP) is a collection of controllable assets, batteries
+  in our case, that we can use to bring additional stability to the grid.
+
+  Our `Vpp` module is simpler than what would be used in the real-world, it can
+  only do two things:
+
+    * `current_power` - calculate the total amount of power all our batteries
+      are contributing to the grid
+    * `export` - export an additional amount of power to the grid
 
-  Our Vpp module is simpler than what would be used in the real-world, it can only do
-  two things. Firstly, it can calculate the total amount of power all of our batteries
-  are contributing to the grid. Secondly, it can be asked to export an additional amount
-  of power to the grid.
   """
 
-  def add_asset(battery) do
+  def add_asset(_battery) do
```

Comment:
Yes, agreed on renaming this!

```suggestion
  def add_battery(_battery) do
```

Reply:
Sounds good! I'll get it updated soon and finish up this PR right after our Dev Summit.